sdk/tables/Azure.Data.Tables/src/TableSharedKeyPipelinePolicy.cs

```diff
@@ -98,7 +97,7 @@ private string BuildCanonicalizedResource(Uri resource)
                     if (name == "comp")
                     {
 #pragma warning disable CA1308 // Normalize strings to uppercase
-                        cr.Append('\n').Append(name.ToLowerInvariant()).Append(':').Append(parameters[name]);
+                        cr.Append('?').Append(name.ToLowerInvariant()).Append('=').Append(parameters[name]);
```

Comment:
What is this doing?

Reply:
Since we are using the [Shared Key Lite format](https://docs.microsoft.com/en-us/rest/api/storageservices/authorize-with-shared-key#shared-key-lite-and-table-service-format-for-2009-09-19-and-later) , the query parameter must be handled differently than standard. This only came to light when I started testing the AccessPolicy methods.